### PR TITLE
Add post about Hacktoberfest 2020

### PR DESCRIPTION
**Pull request by LouiseBonnard**
Add a post to announce Hacktoberfest 2020. Yep, just that.

### CODE REVIEW

File: news/_posts/2020-10-01-hacktoberfest-2020-is-coming.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+As the open source community spreads universal values, everyone is welcome to learn and share knowledge, and therefore stands together to build beneficial technologies. All those individual contributions create one great collective effort that makes this possible.
+
+As a result, it is probably the best place to start involving in the projects you want to support. And the numbers speak for themselves since last year, no less than 483,127 pull requests have been submitted on 154,767 participating repositories!
+
+![Hacktoberfest 2020](/assets/images/2020/09/Hacktoberfest-2020.png)
+
+
+## Get acquainted with the Hacktoberfest challenge
+
+Every contributor achieving this goal of 4 contributions merged will be sent a pack of nice goodies including a tee-shirt and many wonderful stickers. It is the perfect occasion to start a collection! Looking for a more sustainable gift? An environment-friendly option, the ‘pick trees, not tees’ option, will plant a tree instead of delivering you goodies. ;-)
```
Comment (MatShir) [inline news/_posts/2020-10-01-hacktoberfest-2020-is-coming.md:25]: "the 'pick" est-il intentionnel ?

Comment (LouiseBonnard) [reply news/_posts/2020-10-01-hacktoberfest-2020-is-coming.md:25]: Yep, it is a goodie option called _pick trees, not tees_. It lets the participants choose between the classic Hacktoberfest tee-shirt and a more green goodie: a planted tree! 🌳

**Review by matks — APPROVED** [review]
Proof-reading: ✅

File: news/_posts/2020-10-01-hacktoberfest-2020-is-coming.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title:  "Hacktoberfest 2020 is coming"
+subtitle: "Bring the lights on open source projects"
+date:  2020-10-01 09:10:11
+authors: [LouiseBonnard]
+icon: icon-github-alt
+image: /assets/images/2020/09/Hacktoberfest-2020.png
```
Comment (matks) [inline news/_posts/2020-10-01-hacktoberfest-2020-is-coming.md]: Image is maybe a little bit bigger than needed 😁

**Comment by Julievrz** [comment]
> Proof-reading: ✅

I read it too ✅

File: news/_posts/2020-10-01-hacktoberfest-2020-is-coming.md (hunk 2 above)
Comment (LouiseBonnard) [reply news/_posts/2020-10-01-hacktoberfest-2020-is-coming.md]: Oops, you're right, I'll create a banner to have it better sized, thanks for the comment! :-)

Comment (LouiseBonnard) [reply news/_posts/2020-10-01-hacktoberfest-2020-is-coming.md]: @matks, I've just changed the image, I let you check!

**Comment by LouiseBonnard** [comment]
@ttoine, could you publish it tomorrow morning?

**Comment by ttoine** [comment]
thanks @LouiseBonnard

**Pull request merged**